.github/workflows/macos.yml

```diff
@@ -0,0 +1,50 @@
+name: macos
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+  macos:
+    runs-on: macos-latest
+
+    steps:
+
+    - name: Install Codelite's dependencies
+      run: |
+        # brew update && brew upgrade # fails somehow
+        brew install git cmake libssh hunspell
+
+    # WxWidgets
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+          repository: wxWidgets/wxWidgets
+          path: wxWidgets
+          submodules: recursive
+    
+    - name: Build and install wxWidgets
+      run: |
+        mkdir wxWidgets/build-release
+        cd wxWidgets/build-release
+        ../configure --enable-shared --enable-monolithic --with-osx_cocoa CXX='clang++ -std=c++11 -stdlib=libc++ -I../src/tiff/libtiff' CC=clang --disable-debug --disable-mediactrl --enable-stl
+        make -j10
```

Comment:
Why hardcode it to 10 jobs?

Also, CodeLite is now built using C++17, no? Is there any reason why wxWidgets is built using an older standard on macOS (I've never owned a Mac)?

Reply:
@Jarod42 we could probably use

```bash
 make -j$(sysctl -n hw.ncpu) install
``

Reply:
I mostly copy-paste code from https://docs.codelite.org/build/build_wx_widgets/#macos

I don't have MacOS neither.

Created https://github.com/eranif/codelite/pull/3339.